src/lib/definition-parser.ts

```diff
@@ -93,15 +93,21 @@ async function combineDataForAllTypesVersions(
     });
     const allTypesVersions = [dataForRoot, ...dataForOtherTypesVersions];
 
+    interface OptionalPackageJSON { readonly license?: unknown; readonly dependencies?: unknown; peerDependencies?: unknown; }
+
     // tslint:disable-next-line await-promise (tslint bug)
-    const packageJson = hasPackageJson ? await fs.readJson(packageJsonName) as { readonly license?: unknown, readonly dependencies?: unknown } : {};
+    const packageJson = hasPackageJson ? await fs.readJson(packageJsonName) as OptionalPackageJSON : {};
     const license = getLicenseFromPackageJson(packageJson.license);
-    const packageJsonDependencies = checkPackageJsonDependencies(packageJson.dependencies, packageJsonName);
+    const packageJsonDependencies = checkPackageJsonDependencies(packageJson.dependencies, packageJsonName, /* checkWhitelist */ true);
+    const packageJsonPeerDependencies = checkPackageJsonDependencies(packageJson.peerDependencies, packageJsonName, /* checkWhitelist */ false);

```

Comment:
why do peer dependencies get a pass from the whitelist?

Reply:
I originally wrote `"react-native"` in the peerDependencies - and that isn't whitelisted because it exists inside `@types`.

I originally wrote almost the same function, but realized if I could skip the whitelist then it'd be the same code. 

I'm open to the idea that `@types/react-native" should be the bit in peerDependencies instead. Then I'd have to do a bit of work to remove the `@types` when filtering also, but that could be an ok trade-off instead.